src/main/java/com/shade/decima/ui/data/viewer/texture/controls/ImagePanel.java

```diff
@@ -198,19 +222,47 @@ public void fit() {
         }
     }
 
+    public boolean isHDR() {
+        return image != null && image.getData().getTransferType() == DataBuffer.TYPE_FLOAT;
+    }
+
     private void update() {
         if (provider == null) {
             return;
         }
 
         if (image == null) {
             image = provider.getImage(mip, slice);
+
+            if (image.getData().getTransferType() == DataBuffer.TYPE_FLOAT) {
```

Comment:
Can the `applyExposure` perform this check instead? It looks like it should be callee's responsibility, no the caller's.

Additionally, the method `isHDR` can be used instead